backend/src/main/java/com/park/utmstack/web/rest/UtmConfigurationParameterResource.java

```diff
@@ -67,7 +68,7 @@ public ResponseEntity<Void> updateConfigurationParameters(@Valid @RequestBody Li
         try {
             Assert.notEmpty(parameters, "There isn't any parameter to update");
             for (UtmConfigurationParameter parameter : parameters) {
-                if(parameter.getConfParamDatatype().equals("email_list") && parameter.getConfParamRegexp() != null){
+                if(parameter.getConfParamDatatype().equals("email_list") && StringUtils.hasText(parameter.getConfParamRegexp())){
                     Errors errors = new BeanPropertyBindingResult(parameter, "utmConfigurationParameter");
                     emailValidatorService.validate(parameter, errors);
```

Comment:
create a validation factory, where you pass the configuration that changes, and validate, this will be more clear, and we can add more validations to configurations in the future